.env.example

```
@@ -118,3 +118,28 @@ NEXT_PUBLIC_NAME=
 ############################
 # [OPTIONAL] List of allowed domains with app.endatix.com robots.txt behavior (User-agent: *, Allow: /login, Disallow: /). Comma-separated list.
 ROBOTS_ALLOWED_DOMAINS=app.endatix.com
+
+
```

dotenv-linter (4.0.0)

[warning] [ExtraBlankLine] Extra blank line detected

(ExtraBlankLine)